src/App.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import './App.css';
+import Clock from "./components/Clock"
+import Navbar from "./components/Navbar"
+import ActionRow from "./components/ActionRow"
+import AdjustRow from "./components/AdjustRow"
+import MusicRow from './components/MusicRow'
+
+export class App extends Component {
+  constructor(){
+    super()
+    this.state={
+      workTime:{
+        minutes:'25',
+        seconds:'00'
+      },
+      breakTime:{
+        minutes:'05',
+        seconds:'00'
+      },
```

Comment:
The only difference is in the minutes, so you can hold one `MainMinutes`, that will be changed  determined on the `isWorkMode`